Demo/src/main/java/com/paypal/android/ui/PaymentMethodsFragment.kt

```diff
@@ -78,6 +84,10 @@ class PaymentMethodsFragment : Fragment() {
         navigate(PaymentMethodsFragmentDirections.actionPaymentMethodsFragmentToCardFragment())
     }
 
+    private fun launchPayPalButtonFragment() {
```

Comment:
Do we need a separate fragment for the buttons? Could we replace the current demo app PayPal flow button with these new buttons instead?

Reply:
Since PaymentMethodsFragment uses Compose to create the view, I would need to construct my view as Composable too. I already spent some time doing this without success but I agree with the fact of combining these two, so I will figure something out.

Reply:
There's actually interop between Compose and Android views. You should be able to keep the button classes as is and add them to the existing Compose view. See https://developer.android.com/jetpack/compose/interop/interop-apis#views-in-compose

Reply:
@tdchow nice! thanks for the heads up, worked like a charm